CMakeLists.txt

```diff
@@ -15,7 +15,14 @@ endif()
 
 project(jwt-cpp)
 
-option(JWT_BUILD_EXAMPLES "Configure CMake to build examples (or not)" ON)
+# Determine if this is built as a subproject (using add_subdirectory)
+if(CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
```

Comment:
There's a modern way for this https://cmake.org/cmake/help/latest/variable/PROJECT_IS_TOP_LEVEL.html

Not sure when it was added

Reply:
I don't think it's worth bumping the version from 3.14 for this feature alone